src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -160,52 +164,119 @@ We support
+                        {
+                            return new UserAllocation()
+                            {
+                                Variant = userAllocation[ConfigurationFields.AllocationVariantKeyword],
+                                Users = userAllocation.GetSection(ConfigurationFields.UserAllocationUsers).Get<IEnumerable<string>>()
+                            };
+                        }),
+                        Group = allocationSection.GetSection(ConfigurationFields.GroupAllocationSectionName).GetChildren().Select(groupAllocation =>
+                        {
+                            return new GroupAllocation()
+                            {
+                                Variant = groupAllocation[ConfigurationFields.AllocationVariantKeyword],
+                                Groups = groupAllocation.GetSection(ConfigurationFields.GroupAllocationGroups).Get<IEnumerable<string>>()
+                            };
+                        }),
+                        Percentile = allocationSection.GetSection(ConfigurationFields.PercentileAllocationSectionName).GetChildren().Select(percentileAllocation =>
+                        {
+                            return new PercentileAllocation()
+                            {
+                                Variant = percentileAllocation[ConfigurationFields.AllocationVariantKeyword],
+                                From = percentileAllocation.GetValue<double>(ConfigurationFields.PercentileAllocationFrom),
+                                To = percentileAllocation.GetValue<double>(ConfigurationFields.PercentileAllocationTo)
+                            };
+                        }),
+                        Seed = allocationSection[ConfigurationFields.AllocationSeed]
+                    };
+                }
+
+                IEnumerable<IConfigurationSection> variantsSections = configurationSection.GetSection(ConfigurationFields.VariantsSectionName).GetChildren();
+                variants = new List<VariantDefinition>();
+
+                foreach (IConfigurationSection section in variantsSections)
+                {
+                    if (int.TryParse(section.Key, out int _) && !string.IsNullOrEmpty(section[ConfigurationFields.NameKeyword]))
+                    {
+                        VariantDefinition variant = new VariantDefinition()
+                        {
+                            Name = section[ConfigurationFields.NameKeyword],
+                            ConfigurationValue = section.GetSection(ConfigurationFields.VariantDefinitionConfigurationValue),
+                            ConfigurationReference = section[ConfigurationFields.VariantDefinitionConfigurationReference],
+                            StatusOverride = section.GetValue<StatusOverride>(ConfigurationFields.VariantDefinitionStatusOverride)
```

Comment:
Why isn't this enum handled like others?

Reply:
My bad, this should definitely be checked like the previous enums.